website_sale_cart_preview/static/src/js/website_sale_cart_preview.js

```diff
@@ -0,0 +1,45 @@
+$(document).ready(function() {
```

Comment:
@oscarolar that seems to be specific for snippets, for example (excluding the source code file name) the `start` method accepts and `editable_mode` param. It probably works, but are we sure it's the correct way to do this? Do you know if there are any more general-purpose registries that could be used for this?